tests/contrib/django/test_django_snapshots.py

```diff
@@ -156,28 +156,38 @@ def test_psycopg_query_default(client, psycopg2_patched):
 
 
 @pytest.mark.skipif(django.VERSION < (3, 0, 0), reason="ASGI not supported in django<3")
-@pytest.mark.snapshot(
+@snapshot(
     variants={
         "30": (3, 0, 0) <= django.VERSION < (3, 1, 0),
         "31": (3, 1, 0) <= django.VERSION < (3, 2, 0),
         "3x": django.VERSION >= (3, 2, 0),
     },
+    token_override="tests.contrib.django.test_django_snapshots.test_asgi_200",
 )
-def test_asgi_200(daphne_client):
-    resp = daphne_client.get("/")
+@pytest.mark.parametrize("django_asgi", ["application", "channels_application"])
+def test_asgi_200(django_asgi):
+    resp = daphne_client_request(django_asgi, "GET", "/")
     assert resp.status_code == 200
     assert resp.content == b"Hello, test app."
 
 
 @pytest.mark.skipif(django.VERSION < (3, 0, 0), reason="ASGI not supported in django<3")
-@pytest.mark.snapshot(
+@snapshot()
+def test_simple_asgi_app():
```

Comment:
a comment here saying that this shouldn't produce a trace, and why would be nice

Reply:
Here we didn't produce a trace for the simple asgi snapshot test because the url path was set to the application root ( `/` ). I changed the path to `/simple-asgi-app/` and this generate a snapshot with a trace.